### DISCUSSION TITLE
Cross-compilation using CMakeLists.txt

### DISCUSSION THREAD

**Discussion by n1kaaa** [curl/curl#8535] [Q&A]
Hi! I'm trying to build libcurl for ARM64 using CMake but it doesn't seem to be using the right compiler. I can see that the option to cross-compile to build programs to run on HOST is available for the configure script. Is there a way I can set the same using CMakeLists?

**Comment by jzakrzewski** [comment]
Read about CMake toolchain files. You just prepare such a file that describes your toolchain and tell CMake to use it.
